.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*.js]
+indent_style = space
+indent_size = 2
+insert_final_newline = true
+max_line_length = 160
```

Comment:
Personally, I'd go with 100 and fix them as I go. @MarshallOfSound any objections?